reframe/frontend/cli.py

```diff
@@ -523,6 +523,32 @@ def main():
     )
 
     # Options not associated with command-line arguments
+    argparser.add_argument(
+        dest='use_login_shell',
+        envvar='RFM_USE_LOGIN_SHELL',
+        configvar='general/use_login_shell',
+        action='store_true',
+        help='Use a login shell for job scripts'
+    )
+    argparser.add_argument(
+        dest='autodetect_method',
+        envvar='RFM_AUTODETECT_METHOD',
+        action='store',
+        help='Method to detect the system'
+    )
+    argparser.add_argument(
+        dest='autodetect_fqdn',
+        envvar='RFM_AUTODETECT_FQDN',
+        action='store_true',
+        help='Use FQDN as host name'
+    )
```

Comment:
I'd suggest to add a default value here, too.

Reply:
I think with the variables non-associated to command line options, the defaults set through the argparser are ignored. That's why I'm setting the defaults later on the cli.

Reply:
Ok, you are right.